docs/reference/mapping/runtime.asciidoc

```diff
@@ -1,5 +1,7 @@
 [[runtime]]
 == Runtime fields
+beta::[]
```

Comment:
can we make the beta change separately given that it is not related to this specific PR?

Reply:
Yes -- I can remove the beta admonitions and include them in a separate PR.